src/plugins/regions.ts

```diff
@@ -106,7 +109,7 @@ class SingleRegion extends EventEmitter<RegionEvents> {
       pointer-events: all;
     `,
     )
-
+  
```

Comment:
Please run `yarn prettier` next time you commit, it should take care of these spaces.